### ISSUE TITLE
rsync in sample file

### ISSUE THREAD

**Issue by johnoooo** [josenk/vagrant-vmware-esxi#37]
This is a minor issue, but it took me 15 minutes to figure out what is going wrong. 

The sample Vagrant file is a bit misleading when it comes to the `synced_folder` option. The machine is created without a synced folder, as the second line overwrites the first.
```
  #  Use rsync and NFS synced folders. (or disable them)
  config.vm.synced_folder('.', '/vagrant', type: 'rsync')
  config.vm.synced_folder('.', '/vagrant', type: 'nfs', disabled: true)
```

I would go for
```
  #  Use rsync and NFS synced folders. (or disable them)
  #config.vm.synced_folder('.', '/vagrant', type: 'rsync')
  config.vm.synced_folder('.', '/vagrant', type: 'nfs', disabled: true)
```
or
```
  #  Use rsync and NFS synced folders. (or disable them)
  #config.vm.synced_folder('.', '/vagrant', type: 'nfs', disabled: true)
  config.vm.synced_folder('.', '/vagrant', type: 'rsync')
```

This does what I would expected. It rsyncs the local folder to /vagrant.

**Comment by josenk** [comment]
Yup, good point!   I'll update the example Vagrantfiles.

**Comment by josenk** [comment]
fixed in v2.2.2
